docs/cicd_tools/loader.md

```diff
@@ -0,0 +1,40 @@
+# loader module
+
+This module provides functions to help loading the different modules included in this library.
```

Comment:
Can be slimmed down to "Loads other modules provided by the library."

And you should probably add your "internal use only"warning right at the top. So something like:

"Loads other modules provided by the library. This is an internal API for the CICD tools library and not intended for public use. Library users should use the bootstrap.sh script in lieu of calling the loader module itself."

Reply:
Done ! I reworked it using your suggestion:
https://github.com/RedHatInsights/cicd-tools/pull/32/commits/88338c4dfcd9a84b41aeaa576bd362c25b835ab2